parser-parse.go

```diff
@@ -150,7 +151,7 @@ func getParsedDate(dateStr string) *time.Time {
 
 	d, err := dateparse.ParseAny(dateStr)
 	if err != nil {
-		fmt.Printf("Failed to parse date \"%s\"\n", dateStr)
+		log.Printf("Failed to parse date \"%s\"\n", dateStr)
 		return nil
```

Comment:
I was using it to detect unsupported date formats, but I'll delete it.

Reply:
He means switching from `fmt` to `log`, not deleting it. Log line works fine to display errors in console 👍

Reply:
My apologies, I misread the code . I also noticed that the `Parser` has a `log` method that considers `Debug` mode. It might be better to make `getParsedDate` a method of `Parser` struct so we can use the Parser's log method and print the error in Debug mode only. What do you think?